tests/test_updater_validation.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+# Copyright 2022, New York University and the TUF contributors
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+"""Test ngclient Updater validations.
+"""
+
+import os
+import sys
+import tempfile
+import unittest
+
+from tests import utils
+from tests.repository_simulator import RepositorySimulator
+from tuf import unittest_toolbox
+from tuf.ngclient import Updater
+
+
+class TestUpdater(unittest_toolbox.Modified_TestCase):
+    """Test ngclient Updater input validation."""
+
+    def setUp(self) -> None:
+        # pylint: disable-next=consider-using-with
+        self.temp_dir = tempfile.TemporaryDirectory()
+        self.metadata_dir = os.path.join(self.temp_dir.name, "metadata")
+        self.targets_dir = os.path.join(self.temp_dir.name, "targets")
+        os.mkdir(self.metadata_dir)
+        os.mkdir(self.targets_dir)
+
+        # Setup the repository, bootstrap client root.json
+        self.sim = RepositorySimulator()
+        with open(os.path.join(self.metadata_dir, "root.json"), "bw") as f:
+            f.write(self.sim.signed_roots[0])
+
+    def tearDown(self) -> None:
+        self.temp_dir.cleanup()
+
+    def _new_updater(self) -> Updater:
+        return Updater(
+            self.metadata_dir,
+            "https://example.com/metadata/",
+            self.targets_dir,
+            "https://example.com/targets/",
+            fetcher=self.sim,
+        )
+
+    def test_empty_target_path_validation(self) -> None:
+        updater = self._new_updater()
+        target_file = updater.get_targetinfo("")
+        self.assertIsNone(target_file)
```

Comment:
empty string is not a special case anywhere so this is the same test as `test_missing_targetinfo()`?